src/test/java/seedu/bookmark/logic/commands/EditCommandTest.java

```diff
@@ -101,23 +104,24 @@ public void execute_filteredList_success() {
 
     @Test
     public void execute_duplicatePersonUnfilteredList_failure() {
```

Comment:
Could you update the tests names as well

Reply:
Thank you for pointing that out!